WordPress/Classes/Utility/Blogging Reminders/BloggingRemindersStore.swift

```diff
@@ -0,0 +1,306 @@
+import Combine
+
+/// Store for the blogging reminders.  This class should not be used directly.  Please refer to `BloggingReminders` instead.
+///
+class BloggingRemindersStore {
+    /// An enum that describes a user-defined reminder schedule.
+    ///
+    enum Schedule {
+        /// No reminder schedule.
+        ///
+        case none
+
+        /// Weekdays reminders
+        ///
+        case weekdays(_ days: [ReminderScheduleWeekday])
+    }
+
+    enum Weekday {
+        case monday(notificationID: String)
+        case tuesday(notificationID: String)
+        case wednesday(notificationID: String)
+        case thursday(notificationID: String)
+        case friday(notificationID: String)
+        case saturday(notificationID: String)
+        case sunday(notificationID: String)
+
+        var notificationID: String {
+            switch self {
+            case .monday(let notificationID),
+                 .tuesday(let notificationID),
+                 .wednesday(let notificationID),
+                 .thursday(let notificationID),
+                 .friday(let notificationID),
+                 .saturday(let notificationID),
+                 .sunday(let notificationID):
+
+                return notificationID
+            }
+        }
+
+        /// The weekday identifier.
+        ///
+        var identifier: BloggingReminders.Weekday {
+            switch self {
+            case .sunday:
+                return .sunday
+            case .monday:
+                return .monday
+            case .tuesday:
+                return .tuesday
+            case .wednesday:
+                return .wednesday
+            case .thursday:
+                return .thursday
+            case .friday:
+                return .friday
+            case .saturday:
+                return .saturday
+            }
+        }
+    }
+
+    private let dataFileURL: URL?
```

Comment:
Shall we move this down with the other dataFile properties below?

Reply:
The properties for the default store have been removed from this class as part of removing the singletons from these classes.

Can you let me know if those are the properties you were referring to?